<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PairAlign</class>
 <widget class="QMainWindow" name="PairAlign">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>719</width>
    <height>444</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>PCLViewer</string>
  </property>
  <widget class="QWidget" name="central_widget">
   <layout class="QGridLayout" name="grid_layout">
    <item row="0" column="0">
     <layout class="QVBoxLayout" name="vertical_layout">
      <item>
       <layout class="QGridLayout" name="aling_layout">
        <item row="0" column="0">
         <layout class="QGridLayout" name="grid_layout_x">
          <item row="1" column="1">
           <widget class="QLCDNumber" name="number_x">
            <property name="digitCount">
             <number>7</number>
            </property>
            <property name="segmentStyle">
             <enum>QLCDNumber::Flat</enum>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QSlider" name="slider_x">
            <property name="minimum">
             <number>-6400</number>
            </property>
            <property name="maximum">
             <number>6400</number>
            </property>
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
           </widget>
          </item>
          <item row="0" column="0">
           <widget class="QLabel" name="label_x">
            <property name="text">
             <string>X</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item row="1" column="3">
         <layout class="QGridLayout" name="grid_layout_scale">
          <item row="1" column="0">
           <widget class="QSlider" name="slider_scale">
            <property name="minimum">
             <number>1</number>
            </property>
            <property name="maximum">
             <number>20000</number>
            </property>
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
           </widget>
          </item>
          <item row="0" column="0">
           <widget class="QLabel" name="label_scale">
            <property name="text">
             <string>Scale</string>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QLCDNumber" name="number_scale">
            <property name="digitCount">
             <number>7</number>
            </property>
            <property name="segmentStyle">
             <enum>QLCDNumber::Flat</enum>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item row="0" column="1">
         <layout class="QGridLayout" name="grid_layout_y">
          <item row="1" column="1">
           <widget class="QLCDNumber" name="number_y">
            <property name="digitCount">
             <number>7</number>
            </property>
            <property name="segmentStyle">
             <enum>QLCDNumber::Flat</enum>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QSlider" name="slider_y">
            <property name="minimum">
             <number>-4800</number>
            </property>
            <property name="maximum">
             <number>4800</number>
            </property>
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
           </widget>
          </item>
          <item row="0" column="0">
           <widget class="QLabel" name="label_y">
            <property name="text">
             <string>Y</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item row="1" column="2">
         <layout class="QGridLayout" name="grid_layout_yaw">
          <item row="1" column="0">
           <widget class="QSlider" name="slider_yaw">
            <property name="minimum">
             <number>-1800</number>
            </property>
            <property name="maximum">
             <number>1800</number>
            </property>
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QLCDNumber" name="number_yaw">
            <property name="digitCount">
             <number>7</number>
            </property>
            <property name="segmentStyle">
             <enum>QLCDNumber::Flat</enum>
            </property>
           </widget>
          </item>
          <item row="0" column="0">
           <widget class="QLabel" name="label_yaw">
            <property name="text">
             <string>Yaw</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item row="1" column="1">
         <layout class="QGridLayout" name="grid_layout_pitch">
          <item row="0" column="0">
           <widget class="QLabel" name="label_pitch">
            <property name="text">
             <string>Pitch</string>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QLCDNumber" name="number_pitch">
            <property name="digitCount">
             <number>7</number>
            </property>
            <property name="segmentStyle">
             <enum>QLCDNumber::Flat</enum>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QSlider" name="slider_pitch">
            <property name="minimum">
             <number>-1800</number>
            </property>
            <property name="maximum">
             <number>1800</number>
            </property>
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item row="0" column="2">
         <layout class="QGridLayout" name="grid_layout_z">
          <item row="1" column="1">
           <widget class="QLCDNumber" name="number_z">
            <property name="digitCount">
             <number>7</number>
            </property>
            <property name="segmentStyle">
             <enum>QLCDNumber::Flat</enum>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QSlider" name="slider_z">
            <property name="minimum">
             <number>-10240</number>
            </property>
            <property name="maximum">
             <number>10240</number>
            </property>
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
           </widget>
          </item>
          <item row="0" column="0">
           <widget class="QLabel" name="label_z">
            <property name="text">
             <string>Z</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item row="1" column="0">
         <layout class="QGridLayout" name="grid_layout_roll">
          <item row="1" column="0">
           <widget class="QSlider" name="slider_roll">
            <property name="minimum">
             <number>-1800</number>
            </property>
            <property name="maximum">
             <number>1800</number>
            </property>
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
           </widget>
          </item>
          <item row="0" column="0">
           <widget class="QLabel" name="label_roll">
            <property name="text">
             <string>Roll</string>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QLCDNumber" name="number_roll">
            <property name="digitCount">
             <number>7</number>
            </property>
            <property name="segmentStyle">
             <enum>QLCDNumber::Flat</enum>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item row="0" column="3">
         <layout class="QHBoxLayout" name="horizontal_layout">
          <item>
           <widget class="QPushButton" name="save_button">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="minimumSize">
             <size>
              <width>130</width>
              <height>0</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>130</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="text">
             <string>Rotate and Save</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QVTKWidget" name="pcl_widget" native="true">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>300</width>
          <height>300</height>
         </size>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QVTKWidget</class>
   <extends>QWidget</extends>
   <header location="global">QVTKWidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
